seclang/parser.go

```diff
@@ -165,6 +169,14 @@ func (p *Parser) SetCurrentDir(dir string) {
 	p.currentDir = dir
```

Comment:
Should we remove `SetCurrentDir`? The difference between `SetCurrentDir(dir)` and `SetRoot(os.DirFS(dir))` is that `../` to read files outside of `dir` does not work in the latter. It seems that could be a good thing though, let me know what you think.

Reply:
I don't think `..` should be supported once `SetCurrentDir`. It is similar to what happens with docker context.